engine/requirements.in

```diff
@@ -68,3 +68,4 @@ google-auth-oauthlib==1.2.0
 # requests==2.32.3 is installing idna==3.6 but supports idna>=2.5,<4
 # https://github.com/psf/requests/blob/v2.32.3/setup.py#L63
 idna==3.7
+bleach==6.1.0
```

Comment:
I didn't see any existing libraries/methods for sanitizing inputs so decided to go with this